internal/apiserver/graphql/schema.go

```diff
@@ -0,0 +1,148 @@
+package graphql
+
+import (
+	"github.com/astanishevskyi/http-server/internal/apiserver/connectors"
+	"github.com/graphql-go/graphql"
+	"log"
+)
+
+func Schema(grpc connectors.GrpcConnector) (*graphql.Schema, error) {
+	var fields = graphql.Fields{
```

Comment:
hope you don't write it on your own and is just autogenerated code, if not, just generate all graphql code from `.graphql` files